backend/controller/console.go

```diff
@@ -101,7 +101,7 @@ func (c *ConsoleService) GetModules(ctx context.Context, req *connect.Request[pb
 				var jsonRequestSchema string
 				if verbSchema.Request != nil {
 					if requestData, ok := verbSchema.Request.(*schema.Ref); ok {
-						jsonSchema, err := schema.DataToJSONSchema(sch, *requestData)
+						jsonSchema, err := schema.RequestResponseToJSONSchema(sch, *requestData)
```

Comment:
yaasssss +1

Reply:
Now that I said that...should this be called `RequestToJSONSchema` instead of `RequestResponse`? Or are you indicating that it can be used with either?

Reply:
can be used with either!